en/concept/sd_card_layout.md

```diff
@@ -0,0 +1,20 @@
+# PX4 SD Card Layout
+
+The PX4 SD Card is used for storing configuration files, flight logs, mission information etc. 
+
+> **Note** The file system is FAT32.
+
+The directory structure/layout is shown below.
+
+Directory/File(s) | Description
+--- | ---
+/etc/ | Extra config (+ mixers)
```

Comment:
Link to https://dev.px4.io/master/en/concept/system_startup.html#replacing-the-system-startup?

Reply:
Yes!